lib/SiftClient.php

```diff
@@ -132,8 +136,9 @@ private function validateArgument($arg, $name, $type) {
             throw new InvalidArgumentException("${name} cannot be empty.");
     }
 
-    private static function restApiUrl($returnScore) {
-        return self::urlPrefix() . '/events' . ($returnScore ? '?return_score=true' : '');
+    private static function restApiUrl($returnScore, $returnAction) {
+        return self::urlPrefix() . '/events' . ($returnScore ? '?return_score=true' : '') 
+            . ($returnAction ? '?return_action=true' : '');
```

Comment:
If both returnScore and returnAction are true, the url string generated would be:

https://api.siftscience.com/v203/events?return_score=true?return_action=true

Which is invalid. It should be:
https://api.siftscience.com/v203/events?return_score=true&return_action=true